fusor-ember-cli/Brocfile.js

```diff
@@ -20,45 +20,6 @@ var app = new EmberApp({
 // please specify an object with the list of modules as keys
 // along with the exports of each module as its value.
 
-app.import('bower_components/jquery-csv/src/jquery.csv.js');
-app.import('bower_components/patternfly/components/bootstrap/dist/js/bootstrap.min.js');
```

Comment:
All of this bower stuff should master for in master. I have a separate commit SHA called 'DEVELOP' that I cherry-pick each time I develop on ember-cli and then I remove the commit before pushing to master or creating the minified assets.